tests/parser_test.js

```diff
@@ -0,0 +1,97 @@
+var Parser = require('../lib/Parser');
+var PhantomEnvironment = require('../lib/PhantomEnvironment');
+var chai = require('chai');
+var expect = chai.expect;
+var path = require('path');
+var _ = require('lodash');
+
+var uri = 'file://' + path.join(__dirname, '/parser.html');
+
+var env = new PhantomEnvironment({
+    url: uri,
+    screen: {
+        width: 1080,
+        height: 200
+    }
+});
+
+describe('Parser', function () {
+    this.timeout(30000);
```

Comment:
I can decrease it, but you know nobody knows how much time it takes for phantomjs